src/main/resources/hemato_last_update_date.txt

```diff
@@ -0,0 +1 @@
+201905031520
```

Comment:
Why making this specific to the date; if we have to provide another piece of information (an API version for example), we will have to change all this.  I would call the file "hemato_data_info.properties" and the content would look like this:
```
last_updated=201905031520
```
The actual key doesn't matter. 

And then in the code you use to read the file, you can just use Properties.load; and age the date as a string and then parse it. Nice and easy.